cg/meta/workflow/microsalt.py

```diff
@@ -64,15 +64,30 @@ def process(self) -> Process:
 
     def get_case_path(self, case_id: str) -> List[Path]:
         """Returns all paths associated with the case."""
-
         case_obj: models.Family = self.status_db.family(case_id)
         lims_project: str = self.get_project(case_obj.links[0].sample.internal_id)
 
-        return [
+        case_paths = [
             Path(path)
-            for path in glob.glob(f"{self.root_dir}/results/{lims_project}_*", recursive=True)
+            for path in glob.glob(f"{self.root_dir}/results/{lims_project}*", recursive=True)
         ]
 
+        self.verify_case_path_age(case_paths, case_id)
+
+        return case_paths
+
+    def verify_case_path_age(self, case_paths: List[Path], case_id: str):
+        old_date = datetime.now() - timedelta(days=21)
```

Comment:
```suggestion
        due_date = datetime.now() - timedelta(days=ANALYSIS_DUE_DATE)
```
Make 21 a parameter into the functiion to make it more general and call it using a constant instead of `21`